scripts/download-vscode.js

```diff
@@ -50,8 +55,15 @@ downloadAndUnzipVSCode(vscodeVersion)
     // Example:
     // ~/salesforcedx-vscode/packages/salesforcedx-vscode-lwc> npm run test:vscode-integration
     if (vscodeBasePath.indexOf(path.sep + 'packages' + path.sep) !== -1) {
+      console.log('### vscode package install found');
       // Do nothing, vscode is already downloaded and extracted in this package
+
+      const userData = path.join(vscodeBasePath, VSCODE_TEST, USER_DATA);
+      console.log('### userdata: ' + userData);
+      // Remove the previously generated user data to ensure we can run the int tests
+      fs.rmSync(userData, { recursive: true, force: true });
     } else {
+      console.log('### in the else');
```

Comment:
Do you want to keep all of these console.log() statements?  Most look helpful.  Wondering about this one and maybe the userdata one above.

Reply:
Good call.  I'll clean up those two and leave the rest.  They are helpful for seeing the flow of the int test run